desktop/cmp/form/FormField.scss

```diff
@@ -0,0 +1,38 @@
+.xh-form-field {
+  .bp3-form-content {
+    position: relative;
+  }
+
+  .bp3-form-helper-text {
+    font-size: var(--xh-font-size-small-px);
+
+    .xh-form-field-pending {
+      width: 15px;
+      height: 15px;
+    }
+
+    .xh-form-field-error-msg {
+      display: none;
+    }
+  }
+
+  &.xh-form-field-invalid .bp3-form-content {
+    padding: 0 0 calc(var(--xh-font-size-small-px) + var(--xh-pad-half-px));
```

Comment:
Cool!  I hope this is working / providing some value. It's exactly the kind of thing I hoped we could do with these vars.

(Is is OK to add the px values in these calc statements? I'm not well versed on when you need to use the underlying unit-less values.)

Reply:
Yes, perfectly fine (and in many cases necessary) to add the unit with calc. What's really cool is when you mix different units (e.g. `calc(100% - 30px)`)